### PR TITLE
chore(web): make build paths consistent 🎡

### PR DESCRIPTION
**Pull request by mcdurdin**
Update all web component build paths to use ./build/ instead of current variety of ./, ./dist/, ./build/

Does not touch build paths for server components yet (that'll come much later).

@keymanapp-test-bot skip

### CODE REVIEW

File: developer/server/src/win32-tray.ts [hunk 1]
```diff
@@ -81,7 +80,7 @@ module.exports = class Win32Tray {
     });
 
     //console.log('starting tray icon');
-    this.signals.forEach(signal => process.on(signal, this.shutdown.bind(this)));
+    process.on('exit', this.shutdown.bind(this));
```
Comment (mcdurdin) [inline developer/server/src/win32-tray.ts:83]: TSC was having trouble with mismatching types on the `signals` array. Given we only use it in one place, this was easier than trying to tidy it up further for now.

**Review by jahorton — COMMENTED** [review]
Seems fine in isolation.  Will return once I've reviewed more/most? of the PR ladder, especially since multiple builds are reported as broken on _this_ rung.

**Comment by jahorton** [comment]
While I realize that the lexical model compiler isn't strictly a

> [keyman]web component

I would like to point out that it still uses `dist`, not `build`.  As long as this is considered 'fine' and intentional, I'm cool with it... but I suspect that may not be intended?

**Comment by mcdurdin** [comment]
> I would like to point out that it still uses `dist`, not `build`. As long as this is considered 'fine' and intentional, I'm cool with it... but I suspect that may not be intended?

Yeah, I noticed that at one point and then forgot about it.

I think I'll fixup the /developer build paths along with the full developer path reorg rather than in this PR chain if that's okay.

**Comment by jahorton** [comment]
> > I would like to point out that it still uses `dist`, not `build`. As long as this is considered 'fine' and intentional, I'm cool with it... but I suspect that may not be intended?
> 
> Yeah, I noticed that at one point and then forgot about it.
> 
> I think I'll fixup the /developer build paths along with the full developer path reorg rather than in this PR chain if that's okay.

Sure.  Is there an issue or PR where I could drop a note / link to that comment, just to be safe?

**Review by jahorton — APPROVED** [review]

**Comment by mcdurdin** [comment]
> Sure. Is there an issue or PR where I could drop a note / link to that comment, just to be safe?

I've added it as a checklist item in #6525, as that's the master list for all the shuffling that I am doing.

**Pull request merged**